docs/twig-components/method/locale.md

```diff
@@ -1,6 +1,4 @@
 # locale
 
-`locale(localeCode)` is a Twig function to ...
-
-
-Source: Bolt
\ No newline at end of file
+`locale(localeCode)` is a Twig function to Returns the locale for the given `localecode`. Please refer to
+the [locales](https://docs.bolt.cm/4.0/localization/locales) documentation.
```

Comment:
Since this PR is going to be merged into 5.0, links should be updated to link to 5.0.